Resources/config/namer.xml

```diff
@@ -4,31 +4,31 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
-    <services>
-        <service id="vich_uploader.namer_uniqid" class="Vich\UploaderBundle\Naming\UniqidNamer" public="true" />
```

Comment:
Specifically the diff is comparing the inline comment that was added on that line.
I'd prefer to leave the inline comments in the configuration file, so that it is more clear why the services are marked as `public=false` and as to their general purpose.

Reply:
I removed the inline comments. so that the original service definitions align with the new ones in the github diff.